apps/ledger-live-desktop/src/renderer/components/PostOnboardingHub/PostOnboardingActionRow.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useCallback } from "react";
+import { Flex, Icons, Tag, Text } from "@ledgerhq/react-ui";
+import { useTranslation } from "react-i18next";
+import { useHistory } from "react-router-dom";
+import { PostOnboardingActionState, PostOnboardingAction } from "@ledgerhq/types-live";
+
+export type Props = PostOnboardingAction & PostOnboardingActionState;
+
+const PostOnboardingActionRow: React.FC<Props> = props => {
+  const {
+    navigationParams,
+    Icon,
+    title,
+    description,
+    tagLabel,
+    startEvent,
+    startEventProperties,
+    completed,
+  } = props;
+  const { t } = useTranslation();
+  /*const navigation = useNavigation();
+
+  const handlePress = useCallback(() => {
+    if (navigationParams) {
+      navigation.navigate(...navigationParams);
+      startEvent && track(startEvent, startEventProperties);
+    }
+  }, [navigationParams, navigation, startEvent, startEventProperties]);*/
+
+  return (
+    <Flex
+      flexDirection="row"
+      alignItems="center"
+      justifyContent="space-between"
+      backgroundColor="neutral.c30"
+      borderRadius="8px"
+      marginBottom="10px"
+      padding="32px 24px 32px 24px"
```

Comment:
Please use values from the spacing scale here (same for `borderRadius` and `marginBottom`